tests/php/Security/SecurityTest.php

```diff
@@ -214,11 +214,12 @@ public function testLogInAsSomeoneElse()
         );
 
         /* We get a good response */
-        $this->assertEquals($response->getStatusCode(), 200, 'We have a 200 OK response');
-        $this->assertNotNull($response->getBody(), 'There is body content on the page');
+        $this->assertEquals($response->getStatusCode(), 302, 'We have a redirection response');
 
         /* Log the user out */
         Security::setCurrentUser(null);
+
+        $this->autoFollowRedirection = true;
```

Comment:
is this set in `setUp()` or `tearDown()`, then this line is not needed?
To me it seems like a weird side-effect to not capture the previous redirection value and re-set that value here, but instead force to `true`.